ddtrace/contrib/internal/django/patch.py

```diff
@@ -80,7 +80,7 @@
         ),
         use_handler_resource_format=asbool(os.getenv("DD_DJANGO_USE_HANDLER_RESOURCE_FORMAT", default=False)),
         use_legacy_resource_format=asbool(os.getenv("DD_DJANGO_USE_LEGACY_RESOURCE_FORMAT", default=False)),
-        _trace_asgi_websocket=os.getenv("DD_ASGI_TRACE_WEBSOCKET", default=False),
```

Comment:
I recommend using `_get_config` to extract environment variables for telemetry purposes. Example here: https://github.com/DataDog/dd-trace-py/blob/e03d26c6d3c6aab8eb8334741052b240277f92d4/ddtrace/settings/peer_service.py#L24

Reply:
I think it's mainly because the code is older. There were some initiatives around migrating them away but it wasn't done everywhere: https://github.com/DataDog/dd-trace-py/pull/13122 .